src/lib/crypto/kimchi_bindings/js/bindings.js

```diff
@@ -1703,49 +1703,13 @@ var PERMUTS_MINUS_1 = 6;
 // Provides: caml_pasta_fp_proof_evaluations_to_rust
 // Requires: plonk_wasm, caml_fp_vector_to_rust, PERMUTS_MINUS_1, COLUMNS
 var caml_pasta_fp_proof_evaluations_to_rust = function(x) {
-    var w = new plonk_wasm.WasmVecVecFp(COLUMNS);
-    for (var i = 0; i < COLUMNS; ++i) {
-      w.push(caml_fp_vector_to_rust(x[1][i + 1]));
-    }
-
-    var coefficients = new plonk_wasm.WasmVecVecFp(COLUMNS);
-    for (var i = 0; i < COLUMNS; ++i) {
-      coefficients.push(caml_fp_vector_to_rust(x[2][i + 1]));
-    }
-
-    var z = caml_fp_vector_to_rust(x[3]);
-
-    var s = new plonk_wasm.WasmVecVecFp(PERMUTS_MINUS_1);
-    for (i = 0; i < PERMUTS_MINUS_1; ++i) {
-      s.push(caml_fp_vector_to_rust(x[4][i + 1]));
-    }
-
-    var generic_selector = caml_fp_vector_to_rust(x[5]);
-    var poseidon_selector = caml_fp_vector_to_rust(x[6]);
-
-    return new plonk_wasm.WasmFpProofEvaluations(w, coefficients, z, s, generic_selector, poseidon_selector);
+    return x;
 };
```

Comment:
TBH I'm not sure how to review this file. Maybe we should talk about it, there's a lot of made up structures and conventions it seems that I need to be aware of

Reply:
The conversions are no longer necessary, so the functions for doing the conversions become these no-ops.

I guess I'm blocked until you figure out how to review this file.